tests/cypress/e2e/unit_tests/elemental_plugin.spec.ts

```diff
@@ -31,29 +29,40 @@ filterTests(['main', 'upgrade'], () => {
   
     it('Add elemental-ui repo', () => {
       topLevelMenu.openIfClosed();
-      cy.contains('local').click();
-      cy.addHelmRepo({repoName: 'elemental-ui', repoUrl: 'https://github.com/rancher/elemental-ui.git', repoType: 'git'});
+      cy.contains('local')
+        .click();
+      cy.addHelmRepo({repoName: 'elemental-ui',
+        repoUrl: 'https://github.com/rancher/elemental-ui.git',
```

Comment:
For later: maybe better to use a var/const? If the repo is changed at some point.